plugins/modules/cloud/rackspace/rax.py

```diff
@@ -831,12 +832,7 @@ def main():
     if not HAS_PYRAX:
         module.fail_json(msg='pyrax is required for this module')
 
-    service = module.params.get('service')
-
-    if service is not None:
-        module.fail_json(msg='The "service" attribute has been deprecated, '
-                             'please remove "service: cloudservers" from your '
-                             'playbook pertaining to the "rax" module')
+    # this if has been here at least as far back as Ansible 2.3.0
```

Comment:
@felixfontein to be removed, left only for the sake of the PR

Reply:
Hmm, I think it would be better to remove this option in a separate PR. This is something I don't mind backporting to stable-2, but I don't want to backport this to stable-1.

Reply:
(Also it should be mentioned in the changelog.)